QuizApp/Login/LoginViewController.swift

```diff
@@ -74,12 +123,58 @@ class LoginViewController: UIViewController {
         print(name)
         print(pass)
         
-        loginUseCase.checkLogin(name: name, password: pass, router: router)
+        loginUseCase.checkLogin(name: name, password: pass, router: router) { session in
+            let defaults = UserDefaults.standard
+            defaults.set(session.token, forKey: "Token")
+            defaults.set(session.userId, forKey: "UserId")
```

Comment:
It might be better to leave saving to UserDefaults in LoginUseCase like it was before, and only call animateViews() in this completion.